Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.h

```diff
@@ -524,9 +524,11 @@ class WebChromeClient final : public WebCore::ChromeClient {
 
     void addInitialTextAnimationForActiveWritingToolsSession() final;
 
-    void addSourceTextAnimationForActiveWritingToolsSession(const WebCore::CharacterRange&, const String&, CompletionHandler<void(WebCore::TextAnimationRunMode)>&&) final;
+    void addSourceTextAnimationForActiveWritingToolsSession(const WTF::UUID& sourceAnimationUUID, const WTF::UUID& destinationAnimationUUID, bool finished, const WebCore::CharacterRange&, const String&, CompletionHandler<void(WebCore::TextAnimationRunMode)>&&) final;
```

Comment:
ditto about changing the `UUID` in the parameter names to `ID`

Reply:
This is like everywhere. If we want to change it, lets do it at the same time that we change to ObjectIDs.